### PR TITLE
Add Noti 0.1.5

### PR DESCRIPTION
**Pull request by jariz**
Noti is a pushbullet client for macOS. It allows you to see and act upon your Android notifications from macOS.
Homepage: https://noti.center/
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/noti.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'noti' do
+  version '0.1.5'
+  sha256 '6c81afbee1146dffc543af6502b193e55a5b241b5cdd996f43016dce03c8a428'
+
+  # github.com/jariz/Noti was verified as official when first introduced to the cask
+  url "https://github.com/jariz/Noti/releases/download/#{version}/Noti.dmg"
+  appcast 'https://rawgit.com/jariz/Noti-appcast/master/appcast.xml',
```
Comment (vitorgalvao) [inline Casks/noti.rb]: Use `https://github.com/jariz/Noti/releases.atom`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/noti.rb (hunk 1 above)
Comment (jariz) [inline Casks/noti.rb]: Thought it'd make more sense to use the sparkle url because the app uses it as well, but will do 👍

**Comment by jariz** [comment]
Regarding the build check, does it really matter that a brew formula already exists? I thought cask & brew used a separate repo?
Eitherway, the one in brew is not related to my project. I can possibly rename it, but would rather not.

**Comment by vitorgalvao** [comment]
> Regarding the build check, does it really matter that a brew formula already exists?

Only if the apps related, which is not the case here.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**